uml/backend-software-system/input-listener.puml

```diff
@@ -0,0 +1,40 @@
+@startuml input-listener-screen
+package "Input Listener" {
+
+    interface ISender {
+        +sendEvent(event: WindowEvent): void
+    }
+
+    class SocketClientSender {
+        +sendEvent(event: WindowEvent): void
+    }
+
+    ISender <|.. SocketClientSender
```

Comment:
The socket logic will be handled by another component (socket client). You could keep this as a firm for when that component will be interpreted.